src/lib/SX127xDriver/SX127x.cpp

```diff
@@ -455,6 +478,14 @@ uint32_t ICACHE_RAM_ATTR SX127xDriver::GetCurrBandwidth()
 {
   switch (currBW)
   {
+  #if defined(RADIO_SX1272)
+  case SX127x_BW_125_00_KHZ:
+    return 125E3;
+  case SX127x_BW_250_00_KHZ:
+    return 250E3;
+  case SX127x_BW_500_00_KHZ:
+    return 500E3;
+  #else
   case SX127x_BW_7_80_KHZ:
```

Comment:
What if the enum was bulked up with some fake values for the unused BW values?  This will remove the need for an extra def here.

Or if that approach is to janky, use !defined(RADIO_SX1272) around the unused BW?